src/pages/home/report/ReportActionCompose.js

```diff
@@ -230,6 +231,65 @@ class ReportActionCompose extends React.Component {
         return this.props.translate('reportActionCompose.writeSomething');
     }
 
+    /**
+     * Returns the list of IOU Options
+     *
+     * @param {Array} reportParticipants
+     * @returns {Array<object>}
+     */
+    getIOUOptions(reportParticipants) {
+        const iouOptions = [];
+        const participants = _.filter(reportParticipants, email => this.props.myPersonalDetails.login !== email);
+        const hasExcludedIOUEmails = lodashIntersection(reportParticipants, CONST.EXPENSIFY_EMAILS).length > 0;
+        const hasMultipleParticipants = participants.length > 1;
+
+        if (hasExcludedIOUEmails || participants.length === 0 || !Permissions.canUseIOU(this.props.betas)) {
+            return iouOptions;
+        }
+
+        if (hasMultipleParticipants) {
+            iouOptions.push(
+                {
+                    icon: Expensicons.Receipt,
+                    text: this.props.translate('iou.splitBill'),
+                    onSelected: () => {
+                        Navigation.navigate(
+                            ROUTES.getIouSplitRoute(
+                                this.props.reportID,
+                            ),
+                        );
+                    },
+                },
+            );
+        } else {
+            iouOptions.push({
```

Comment:
```suggestion
            const iouOptions = [];
            iouOptions.push({
```

Reply:
else statement is removed as it was not needed and variable is declared at the top of the function.